### ISSUE TITLE
[Accessibility- Script 7.3] Allow the site navigation control by the keyboard

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#2909]
**Environment:** https://whitepaper-ft.meeds.io/

**Step of reproduction:**
Access to the site navigation by the keyboard

**Current behavior:**

- Press the enter key => We can't access to the site navigation drawer 

- Access to the site navigation drawer then tabbing through the keyboard => we can't access to subnodes and to actions associated to each node (in the minimized or expanded window)

- the close button does not work when activating it with the enter key

**Expected behavior:**

- Verify that all actions can be triggered when pressing ENTER key and SPACE key.

- Verify that all elements on which a JavaScript event handler is implemented (click, focus, mouseover, blur, keydown, touch…) must be accessible using the keyboard

![Image](https://github.com/user-attachments/assets/a31926da-9f95-4adb-905c-09d2e9319a77)

**Comment by mbensalem** [comment]
Still reproduced on 7.2.x
